client/src/js/services/DebtorGroupService.js

```diff
@@ -35,6 +35,10 @@ function DebtorGroupService($http, util) {
   * @description This function is responsible for create new debtor group
   */
   function create(debtorGroup) {
+
+    // augment object with session values 
+    debtorGroup.enterprise_id = SessionService.enterprise.id;
```

Comment:
With this pattern, we now have three ways of retrieving this information to put in the database:
 1. In the controller, before the service is called to create the group
 2. Here, in the service, before submitting to the server
 3. On the server, using `req.session`,  regardless of whether the client submitted the data or not.

I am a fan of **3**, though I am willing to hear arguments why another should be done.  What do you think?